.github/workflows/push.yml

```
@@ -161,13 +162,19 @@ jobs:
     name: iOS Build and Relaese
     runs-on: macos-latest
     needs: Flutter-Testing
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)